resources/views/messages/welcome.blade.php

```diff
@@ -1 +1,3 @@
-{!! $content['body'] !!}
+<p>{!! $content['body'] !!}</p>
```

Comment:
No it's not markdown, i need to `<p></p>` tags to make sure the body and the unsubscribe link show up on different lines.